src/dumpContent.ts

```diff
@@ -88,6 +95,15 @@ function dumpManifest(objIds: string[]) {
   )
 }
 
+function preprocessObjData(objData: ObjData): ObjData {
```

Comment:
* [ ] FYI Naming: For the use-case at hand instead of the generic-sounding "preprocess" we could also name it more specifically like `clearSensitiveData`, `removePrivateData`, `ignorePerInstanceData`.